lib/node_modules/@stdlib/math/base/special/sincos/manifest.json

```
@@ -0,0 +1,80 @@
+{
+  "options": {
+    "task": "build"
+  },
+  "fields": [
+    {
+      "field": "src",
+      "resolve": true,
+      "relative": true
+    },
+    {
+      "field": "include",
+      "resolve": true,
+      "relative": true
+    },
+    {
+      "field": "libraries",
+      "resolve": false,
+      "relative": false
+    },
+    {
+      "field": "libpath",
+      "resolve": true,
+      "relative": false
+    }
+  ],
+  "confs": [
+    {
+      "task": "build",
+      "src": [
+        "./src/main.c"
+      ],
+      "include": [
+        "./include"
+      ],
+      "libraries": [],
+      "libpath": [],
+      "dependencies": [
+        "@stdlib/math/base/special/rempio2",
```

I believe we need these additional dependencies here due to their presence in `addon.c`:

```suggestion
        "@stdlib/napi/argv",
        "@stdlib/napi/argv-double",
        "@stdlib/napi/argv-float64array",
        "@stdlib/math/base/special/rempio2",
        "@stdlib/napi/export",
```

---

@Planeshifter I've made the changes.